### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by Duckle29** [ai03-2725/random-keyboard-parts.pretty#3]
Hi, I'd like to include this library in a kicad template I use for keyboard projects, but as it doesn't have a license I can't, would you be willing to add a license file clarifying such uses? Thanks! :)

**Comment by Soundtoxin** [comment]
It's been a while. Any chance of adding a license now?

**Comment by antaxiom** [comment]
I agree, I'm currently using this on a product and I need some form of license for it. It would be useful if the owner responded. I'd like to note that ai03 frequently puts the MIT license in their projects.
